### PR TITLE
ODIM-5413 fix event service unit test cases

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

File: svc-events/evcommon/common_test.go [hunk 1]
```diff
@@ -546,9 +395,9 @@ func TestGetPluginStatusandStartUP(t *testing.T) {
 		PluginType:        "RF-GENERIC",
 	})
 	searchKey := GetSearchKey("10.4.1.2", evmodel.DeviceSubscriptionIndex)
-	deviceSubscription, err := evmodel.GetDeviceSubscriptions(searchKey)
+	deviceSubscription, err := st.GetDeviceSubscriptions(searchKey)
 	if err != nil {
 		t.Fatalf("error: %v", err)
 	}
-	assert.Equal(t, "/redfish/v1/EventService/Subscriptions/2", deviceSubscription.Location, "should be same")
+	assert.Equal(t, "https://10.24.1.2/EventService/Subscriptions/1", deviceSubscription.Location, "should be same")
```
Comment (shivacharanms) [inline svc-events/evcommon/common_test.go]: Change to another example IP

File: svc-events/evcommon/common_test.go [hunk 2]
```diff
@@ -546,9 +395,9 @@ func TestGetPluginStatusandStartUP(t *testing.T) {
 		PluginType:        "RF-GENERIC",
 	})
 	searchKey := GetSearchKey("10.4.1.2", evmodel.DeviceSubscriptionIndex)
```
Comment (jeevan-kamkar) [inline svc-events/evcommon/common_test.go]: change the ip

File: svc-events/events/deletesubscription_test.go [hunk 3]
```diff
@@ -611,27 +227,12 @@ func TestDeleteFabricsSubscription(t *testing.T) {
...
-		Auth:          mockIsAuthorized,
-		ContactClient: mockContactClient,
+	encryptedData, err := evcommon.GetEncryptedKey([]byte("testData"))
+	if err != nil {
+		t.Fatalf("%v", err.Error())
 	}
-
-	encryptedData := getEncryptedKey(t, []byte("testData"))
 	target := evmodel.Target{
 		ManagerAddress: "10.4.1.2",
```
Comment (jeevan-kamkar) [inline svc-events/events/deletesubscription_test.go]: change the ip

File: svc-events/events/evtsubscription_test.go [hunk 4]
```diff
@@ -221,8 +62,8 @@ func TestCreateEventSubscription(t *testing.T) {
 	resp := p.CreateEventSubscription(taskID, sessionUserName, req)
 	assert.Equal(t, http.StatusCreated, int(resp.StatusCode), "Status Code should be StatusCreated")
 
-	// test with different event Types and same destinations
-	SubscriptionReq["EventTypes"] = []string{"Alert", "StatusChange"}
+	// try to subscrie with already subscribed destinations
+	SubscriptionReq["Destination"] = "https://10.24.1.15:9090/events"
```
Comment (jeevan-kamkar) [inline svc-events/events/evtsubscription_test.go]: change the ip

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
address the requested changes

File: svc-events/evcommon/common_test.go (hunk 1 above)
Comment (irappa-pattar) [reply svc-events/evcommon/common_test.go]: updated

**Review by jeevan-kamkar — APPROVED** [review]

File: svc-events/events/deletesubscription_test.go (hunk 3 above)
Comment (irappa-pattar) [reply svc-events/events/deletesubscription_test.go]: changed

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**